cmd/vic-machine/upgrade/upgrade.go

```diff
@@ -59,6 +59,12 @@ func (u *Upgrade) Flags() []cli.Flag {
 			Usage:       "Time to wait for upgrade",
 			Destination: &u.Timeout,
 		},
+		cli.IntFlag{
+			Name:        "rollback",
+			Value:       0,
+			Usage:       "Rollback n versions, e.g. --rollback=1 to go to the version before the last upgrade",
```

Comment:
This was in original design with an eye towards the future; actually already had it swapped to a `bool` locally :)